inst/templates/ad_advs.R

```diff
@@ -247,83 +241,90 @@ advs <- advs %>%
     ))
   )
 
-# Add Current HEIGHT Temporary variable (in cm)
-## Calculate Current HEIGHT/LENGTH at each time point Temporary variable----
-hgttmp <- advs %>%
-  filter((PARAMCD == "HEIGHT" | PARAMCD == "LENGTH") & VSSTRESU == "cm") %>%
-  select(STUDYID, USUBJID, HGTTMP = AVAL, HGTTMPU = VSSTRESU, AVISIT)
-
+# Derive Current HEIGHT/LENGTH at each time point Temporary variable----
 advs <- advs %>%
   derive_vars_merged(
-    dataset_add = hgttmp,
-    by_vars = exprs(STUDYID, USUBJID, AVISIT)
+    dataset_add = advs,
+    by_vars = c(get_admiral_option("subject_keys"), exprs(AVISIT)),
+    filter_add = PARAMCD == "HEIGHT" & VSSTRESU == "cm",
```

Comment:
Should we use `toupper()` here and "CM"?  as might be a bit more robust as i imagine most companies map these in uppercase

Reply:
implemented